test/cases/fixes/incompleteTypes/reactTypes/reactPropsFromPropTypes/all/original.tsx

```diff
@@ -75,4 +75,34 @@ import PropTypes from "prop-types";
 		string: PropTypes.string,
 		stringRequired: PropTypes.string.isRequired,
 	};
+
+	class Greeting extends React.Component {
+		render() {
+			return <h1>Hello, {this.props.name}</h1>;
+		}
+	}
+
+	Greeting.propTypes = {
+		name: PropTypes.string,
+	};
+
+	// TODO: The mutation for this is incorrect
+	function HelloWorldComponent({ name }) {
+		return <div>Hello, {name}</div>;
+	}
+
+	HelloWorldComponent.propTypes = {
+		name: PropTypes.string,
+	};
+
+	// TODO: The mutation for this is incorrect
```

Comment:
[Docs] For this and the other added `// TODO:`s, are issue(s) filed? If not, could you file one/them please?